src/fsharp/TcGlobals.fs

```diff
@@ -1227,6 +1227,7 @@ type public TcGlobals(compilingFslib: bool, ilg:ILGlobals, fslibCcu: CcuThunk, d
   member val attribs_Unsupported = [
         tryFindSysAttrib "System.Runtime.CompilerServices.ModuleInitializerAttribute"
         tryFindSysAttrib "System.Runtime.CompilerServices.CallerArgumentExpressionAttribute"
+        tryFindSysAttrib "System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute"
```

Comment:
This is a bit subtle with regard to performance. Could you factor out `attribs_Unsupported` to a `let v_unsupportedAttributes = ...` please to make it clearer?

Reply:
Should all `member val`s be refactored to `let` because we have to avoid "subtle performance"? `member val` should be known as init-once-only.

Reply:
Only if involving a complex mult-line expression, especially something that involved pipelining or is unusual.  I just had to think twice about what we had here